terraform-aws-redshift-cluster/variables.tf

```diff
@@ -1,7 +1,13 @@
 ## Variables file
 
-/* variable "namespace" {
-  description = "(Optional) - Namespace that will be used for the configuration"
+variable "region" {
+  description = "The region to deploy the redis-cluster in"
   type        = string
-  default     = null
-} */
+  default     = "ap-southeast-2"
+}
+
+variable "secondary_region" {
+  description = "The region to deploy the redis-cluster in"
+  type        = string
+  default     = "ap-south-1"
```

Comment:
this isn't a valid region is this used

Reply:
This is the Mumbai region ap-south-1. I had overriden all this in my auto tf vars to make it different regions. i can change this to some other region. Wanted to retain sydney based configs as you had set in other folders